### PR TITLE
Small Updates

### PR DESCRIPTION
**Pull request by pedrogiampietro**
This PR introduces a series of improvements and adjustments mainly focused on refactoring components to use Chakra UI and general improvements to account management and related functionalities. Below are the main points covered:

**### Updates**
Docker: Docker implementation and related schema adjustments.
### **Account Management:**
Adjustments to account management to support two-factor authentication (2FA).
Refactoring the lost account recovery process.
Router: Added 2FA verification on router.
### **Highscores:**
Refactoring to use Chakra UI.
Migration to TypeScript (tsx).
Tables: Correction of table names.
### **APIClient:**
Adjustments to accept multipart.
Implementation of parameters for requests.
### **Guild:**
Refactoring of the guild list and guild page.
Adjustments to the expel members functionality.
Added image to the guild.
Download Page: Refactoring of the download page.
MercadoPago and Shopping: Necessary adjustments made.
Administrative Panel: Implementation of the administrative panel and news section.
### **Other Additions:**
Creation of a "shared" directory to share interfaces.
Added active dropdown functionality.
Saving the creation in the registry.
Thank you in advance for the review and I am open to feedback and suggestions.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,6 +1,14 @@
...
-DATABASE_URL=mysql://root:@localhost:3306/shibaac
+
+MYSQL_PORT=3306
+MYSQL_ROOT=root
+MYSQL_DATABASE=test_db
+MYSQL_PASSWORD=secret
+MYSQL_ROOT_PASSWORD=secret
+MYSQL_HOST=localhost
+
+DATABASE_URL="mysql://root:secret@localhost:3306/test_db"
```
Comment (nekiro) [inline .env.example:11]: is this duplicated? or why is there two database credentials types

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+version: '3.9'
+
+services:
+  database:
```
Comment (nekiro) [inline docker-compose.yml:4]: I think you missed container for starting up the aac itself, cause the docker-compose should contain ready to use aac environment

File: src/layout/Footer.jsx [hunk 3]
```diff
@@ -10,7 +10,7 @@ const Footer = () => {
       padding="10px"
       textAlign="center"
     >
-      Copyright © 2021-2022 Nekiro
+      Copyright © 2021-2022 Nekiro, Pedrog
```
Comment (nekiro) [inline src/layout/Footer.jsx]: Update year

File: src/lib/nodemailer.ts [hunk 4]
```diff
@@ -0,0 +1,35 @@
+import nodemailer from 'nodemailer';
+
+const transporter = nodemailer.createTransport({
+  service: 'hotmail.com',
```
Comment (nekiro) [inline src/lib/nodemailer.ts]: should be configurable in .env all of this smtp stuff

File: src/lib/nodemailer.ts [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    const mailOptions: MailOptions = {
+      from: 'your email',
+      to: email,
+      subject: subject,
+      text: message,
+    };
+
+    await transporter.sendMail(mailOptions);
+  } catch (error: any) {
+    console.error(`Erro ao enviar e-mail: ${error}`);
```
Comment (nekiro) [inline src/lib/nodemailer.ts]: Please english!

File: src/lib/nodemailer.ts [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+}
+
+export async function sendEmail(
+  email: string,
+  subject: string,
+  message: string,
+): Promise<void> {
+  try {
+    const mailOptions: MailOptions = {
+      from: 'your email',
```
Comment (nekiro) [inline src/lib/nodemailer.ts]: configurable too

File: src/prisma/migrations/20230921001241_/migration.sql [hunk 7]
```diff
@@ -0,0 +1,339 @@
+-- CreateTable
+CREATE TABLE `account_bans` (
```
Comment (nekiro) [inline src/prisma/migrations/20230921001241_/migration.sql]: I dont think server tables should be a migration of AAC, we should be migrating only what we need for the aac to work, which does not include server tables

File: src/shared/interfaces/IPlayer.ts [hunk 8]
```diff
@@ -0,0 +1,64 @@
+export interface PlayerData {
```
Comment (nekiro) [inline src/shared/interfaces/IPlayer.ts]: I'm honestly not a fan of naming the interfaces with `I` in the start

File: .env.example (hunk 1 above)
Comment (pedrogiampietro) [reply .env.example:11]: no, it's actually the credentials for docker.

File: docker-compose.yml (hunk 2 above)
Comment (pedrogiampietro) [reply docker-compose.yml:4]: In fact, I'm just running Docker, to have a database, it's functional, but in the future we can add scripts to run the backend, such as migrations, seeds, etc.

**Pull request merged**